.github/workflows/continuous-integration.yml

```diff
@@ -423,9 +423,62 @@ jobs:
       - name: Annotate Stylelint results
         if: ${{ always() }}
         run: node ./script/github-actions/stylelint-annotation-format.js
+  
+  check-app-dependency-graph:
```

Comment:
Do we need to run this on `master` builds? Can we add a condition to run only on non-master branches?

Reply:
Done.